packages/frontend/src/app/utils.ts

```diff
@@ -54,6 +54,19 @@ function checkURL(url: string): boolean {
   }
 }
 
+function checkBatchConfig(batchSize: number, numberOfBatches: number): boolean {
```

Comment:
You've made Nick Wu's favourite joke of if (true) then false; - can concat this to a single condition that is returned